cli/print/channel_attributes.go

```diff
@@ -1,22 +1,33 @@
 package print
 
 import (
+	"github.com/replicatedhq/replicated/pkg/types"
 	"text/tabwriter"
 	"text/template"
-
-	channels "github.com/replicatedhq/replicated/gen/go/v1"
 )
 
-var channelAttrsTmplSrc = `ID:	{{ .Id }}
+var channelAttrsTmplSrc = `ID:	{{ .ID }}
```

Comment:
uhhhhh so apparently this breaks a bunch of tests...?

Reply:
ahh no i think we changed some channels.AppChannel to types.Channel, need to dig into that. stay tuned